.gitignore

```diff
@@ -7,4 +7,5 @@ venv/
 .pytest_cache/
 **__pycache__/
 *.pyc
-app/db.sqlite3
\ No newline at end of file
+app/db.sqlite3
+*.sqlite3
```

Comment:
Empty line is missing. It is a nit and I won't reject it now, but remember for future)

Reply:
Thank you,  will be more carefully